tests/test_lab_evaluate.py

```diff
@@ -130,7 +130,7 @@ def run_mt_bench(cli_runner, error_rate):
             {round(error_rate, 2)}
             """
         )
-    assert result.output == expected
+    assert expected.endswith(result.output)
```

Comment:
This is a bit worrying that undesired output will sneak in after this change.  You could pass --gpus to avoid the extra message instead?

Reply:
I've finally revisited this PR and was able to revert this back to what it was originally